app/client/package.json

```diff
@@ -65,7 +65,10 @@
     "parserOptions": {
       "ecmaVersion": 2020
     },
-    "rules": {},
+    "rules": {
+      "no-shadow": "off",
+      "@typescript-eslint/no-shadow": ["error"]
```

Comment:
Any particular reason to turn this rule off? It's quite a sensible one to avoid ambiguity usually.

Reply:
I was getting an error (`already declared in the upper scope`) after introducing the enum in `types.ts`. This seems to be a [common issue](https://github.com/typescript-eslint/typescript-eslint/issues/2552#issuecomment-691694839), and the only solution I found was to disable the rule completely. This was even suggested in the [typescript-eslint documentation](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-shadow.md#how-to-use).